appveyor.yml

```diff
@@ -58,6 +68,53 @@ on_failure:
 artifacts:
   - path: dist\*
 
+# scripts to run before deployment
+before_deploy:
+  # install gravitybee/generate the standalone executable
+  - mkdir C:\venv
+  - C:\Python36-x64\Scripts\virtualenv C:\venv
+  - C:\venv\Scripts\activate
+  - python --version
+  - pip install gravitybee satsuki
+  - pip install --editable .
+  - gravitybee --src-dir src --sha file --extra-data static --verbose
+  # prepare deployment
+  - set SATS_TAG=$gb_pkg_ver
+  - set SATS_BODY=* [$gb_pkg_name v$gb_pkg_ver CHANGELOG](https://github.com/plus3it/$gb_pkg_name_lower/blob/$gb_pkg_ver/CHANGELOG.md) (Win)
+  - set SATS_REL_NAME=Release v$gb_pkg_ver
+  - set LOCAL_STAGING_DIR=s3_uploads
+  - ps: >-
+      if($env:APPVEYOR_REPO_BRANCH -eq "master") {
+        $env:S3_BUCKET=$env:RELEASE_BUCKET
+      } else {
+        $env:S3_BUCKET=$env:DEV_BUCKET
+      }
+  - set S3_KEYFIX=releases
+  - gravitybee-environs.bat
+  # upload & rename artifacts to appveyor's magic artifact depot in the sky (DITS)
+  - appveyor PushArtifact %GB_ENV_GEN_FILE_W_PATH% -FileName %GB_ENV_APP_VERSION%\%GB_ENV_GEN_FILE%
```

Comment:
this block, where directory names and file names/paths are being contructed from numerous assorted ENVs, is really hard to read and comprehend what is going on... maybe set some well-named ENVs instead of stringing together all these different values? or rethink what gravitybee is outputting so this is easier to handle?

Reply:
Yeah, I'm attacking the same problem two different ways from two different iterations. It's confusing. I'll clean it up.